icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -97,34 +97,6 @@
 #
 #    $TOOLS_ROOT/cldr/cldr-to-icu/build-icu-data.xml
 #
-# The files used in previous versions are the following. These are not used in
-# the ICU-67-and-later process, but for now they should be kept up to date to
-# enable parallel use of the older build process for verification.
-#
-#    $ICU4C_DIR/source/data/icu-config.xml - Update <locales> to add or remove
-#                CLDR locales for inclusion in ICU. Update <paths> to prefer
-#                alt forms for certain paths, or to exclude certain paths; note
-#                that <paths> items can only have draft or alt attributes.
-#
-#                Note that if a language-only locale (e.g. "de") is included in
-#                <locales>, then all region sublocales for that language that
-#                are present in CLDR data (e.g. "de_AT", "de_BE", "de_CH", etc.)
-#                should also be included in <locales>, per PMC policy decision
-#                2012-05-02 (see http://bugs.icu-project.org/trac/ticket/9298).
-#
-#    $ICU4C_DIR/source/data/build.xml - If you are adding or removing break
-#                iterators, you need to update  <fileset id="brkitr" ...> under
-#                <target name="clean" ...> to clean the correct set of files.
-#
-#                If there are new CLDR resource bundle types, you may need to
-#                updated the <remapper> sections to put these in the correct
-#                data subfolder for ICU.
-#
-#    $ICU4C_DIR/source/data/xml/      - If you are adding a new locale, break
-#                iterator, collation tailoring, or rule-based number formatter,
-#                you may need to add a corresponding xml file in (respectively)
-#                the main/, brkitr/, collation/, or rbnf/ subdirectory here.
```

Comment:
In the new tools, where is the correspondence now established between a particular locale and break type (such as locale "ja" with break type 'line alt="loose"') with a particular rules file (like line_loose_cj.txt source, or line_loose_cj.brk compiled)?

Reply:
Hmm, this is not something I know anything about. It's not something which was encoded anywhere in the CLDR tooling I converted. Is it part of the CLDR-to-ICU conversion or a post-processing step?

Can you show me an example of the difference this setting would produce?